CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ option(IREE_BUILD_DOCS "Builds IREE docs." OFF)
 option(IREE_BUILD_SAMPLES "Builds IREE sample projects." ON)
 option(IREE_BUILD_DEBUGGER "Builds the IREE debugger app." OFF)
 option(IREE_BUILD_PYTHON_BINDINGS "Builds the IREE python bindings" OFF)
+option(IREE_BUILD_JAVA_BINDINGS "Builds the IREE java bindings." OFF)
```

Comment:
Can you also document this here: https://github.com/google/iree/blob/main/docs/get_started/cmake_options_and_variables.md#iree_build_python_bindingsbool ?

Reply:
Fixed